utils/copyright_variables.js

```diff
@@ -1,30 +1,57 @@
-export const panelCopy = {
-    title: 'our 3D communication and interaction platform is the solution for Associations with a distributed member base and operations'
-}
-
-export const traditionCopy = {
-    title: 'Are you looking to:',
-    list: [
-        'Amplify member engagement',
-        'Boost retention',
-        'Attract new members'
-    ]
-}
-
-export const solutionCopy = {
-    headerLeft: 'We’re redefining what it means to meet up virtually',
-    headerRight: 'All it takes is your laptop',
-    list: [
-        'Immersive 3D environments',
-        'Dynamic spatial audio',
-        'High-quality video'
-    ],
+export const associationCopy = {
+    panel: {
+        title: 'our 3D communication and interaction platform is the solution for Associations with distributed members and operations'
```

Comment:
change to "Built for Associations to boost member value by offering a better way to network"